compose-profiles/dev-local-3nodes/docker-compose.yml

```diff
@@ -45,11 +46,16 @@ services:
       - I2B2_LOGIN_PASSWORD=e2etest
       - LOG_LEVEL=5
       - MEDCO_NODE_IDX=0
-      - GA_DB_NAME=gamedcosrv0
+      - MC_DB_NAME=medcoconnectorsrv0
       - OIDC_JWKS_URLS=http://keycloak:8080/auth/realms/master/protocol/openid-connect/certs,https://medco-demo.epfl.ch/auth/realms/master/protocol/openid-connect/certs
       - OIDC_JWT_ISSUERS=http://localhost/auth/realms/master,https://medco-demo.epfl.ch/auth/realms/master
       - OIDC_CLIENT_IDS=medco,medco
       - OIDC_JWT_USER_ID_CLAIMS=preferred_username,preferred_username
+      - I2B2_DB_HOST=postgresql
```

Comment:
Those don't all need to be redeclared here since they were already declared in the docker-compose-definitions.yml file. Just keep the one that is different: `I2B2_DB_NAME`.